austinguitest.py

```diff
@@ -322,15 +335,15 @@ def gameplay(self):
 
     def gameplayBase(self):
         if(self.controller.controllerGetPuzzleState() == 1):
-            userinput = simpledialog.askstring("Would you like to save your game?",  "Would you like to save? yes/no:")
-            if userinput:
-                if userinput.lower() == "yes":
+            answer = messagebox.askyesno("Would you like to save?", "Would you like to save the game?")
+            if answer == True:
                     fileName = simpledialog.askstring("Please enter a file name", "File name: ")
                     self.controller.controllerSaveGame(fileName)
-                else:
+            elif answer == False:
                     messagebox.showinfo("Generating a new puzzle", "Time to generate a new puzzle!")
             else:
-                messagebox.showinfo("No information provided", "going to generate a new puzzle!")
+                messagebox.showinfo("Error", "Choose yes or no!")
```

Comment:
Here again I'm confused if their only two options are yes or no why would there be an error if they clicked no (which is what i assume false is?)

Reply:
for both of your comments I think I just forgot to remove it after changing it